source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java

```diff
@@ -1014,43 +1014,70 @@ public ArrayList<String> getSqlDocumentation() {
         b.append(",('testcaseexecution','BrowserFullVersion','','en','Browser Version','This is the full version information of the browser that was used to run the <code class=\\'doc-crbvvoca\\'>test case</code> (only used if that was a GUI application <code class=\\'doc-crbvvoca\\'>test case</code>).',NULL)");
+        b.append(",('testcaseexecution','BrowserFullVersion','','fa','نسخه مرورگر','این نسخه کامل مرورگری است که برای اجرای  <code class=\\'doc-crbvvoca\\'>مورد آزمون</code> مورد استفاده قرار گرفته است.(فقط در صورتی استفاده می شود که یک  <code class=\\'doc-crbvvoca\\'>مورد آزمون</code> مربوط به برنامه دارای رابط گرافیکی باشد.).',NULL)");
         b.append(",('testcaseexecution','BrowserFullVersion','','fr','Version du navigateur',NULL,NULL)");
         b.append(",('testcaseexecution','Build','','en','Sprint','Name of the Build/sprint.',NULL)");
+        b.append(",('testcaseexecution','Build','','fa','اسپرینت','نام ساخت/اسپرینت.',NULL)");
         b.append(",('testcaseexecution','ControlMessage','','en','ControlMessage','This is the message reported by Cerberus on the execution of the <code class=\\'doc-crbvvoca\\'>test case</code>.',NULL)");
+        b.append(",('testcaseexecution','ControlMessage','','fa','پیام کنترل','این پیامی است که توسط Cerberus برای اجرا <code class=\\'doc-crbvvoca\\'>مورد آزمون</code> گزارش کرده است.',NULL)");
         b.append(",('testcaseexecution','controlstatus','','en','RC','This is the return code of the Execution. It can take the following values :<br><br><b>OK</b> : The <code class=\\'doc-crbvvoca\\'>test case</code> has been executed and everything happened as expected.<br><b>KO</b> : The <code class=\\'doc-crbvvoca\\'>test case</code> has been executed and reported an error that will create a bug.<br><b>NA</b> : The <code class=\\'doc-crbvvoca\\'>test case</code> has been executed but some data to perform the test could not be collected (SQL returning empty resultset).<br><b>FA</b> : The <code class=\\'doc-crbvvoca\\'>test case</code> failed to execute because there were an error inside the test such as an SQL error. The <code class=\\'doc-crbvvoca\\'>test case</code> needs to be corrected.<br><b>CA</b> : The <code class=\\'doc-crbvvoca\\'>test case</code> has been cancelled. It failed during the execution because of technical issues (ex. Lost of connection issue to selenium during the execution)<br><b>PE</b> : The execution is still running and not finished yet or has been interupted.',NULL)");
-        b.append(",('testcaseexecution','crbversion','','en','Cerberus Version','This is the version of the Cerberus Engine that executed the <code class=\\'doc-crbvvoca\\'>test case</code>.<br>This data has been created for traceability purpose as the behavious of Cerberus could varry from one version to another.',NULL)");
```

Comment:
Why was that line removed ? Can you put it back ?

Reply:
hi dear vertigo, maybe when i highlighted the line cause the deletion. because i copy the line to bottom then i translate to persian, i apologize

Reply:
fixed